e2e/validation.go

```diff
@@ -102,5 +116,16 @@ func commonLiveVMValidators() []*LiveVMValidator {
 				"cloud-config.txt",
 			},
 		),
+		{
+			Description: "check that curl to wireserver fails",
+			Command:     "curl 'http://168.63.129.16:32526/vmSettings' --connect-timeout 4",
```

Comment:
do we actually need the single quotes?

Reply:
Not for the URL, will remove. But they are needed for headers in the second check (which i will add shortly too)